lldb/docs/resources/debugging-lldb.rst

```diff
@@ -0,0 +1,254 @@
+Debugging LLDB
+==============
+
+This page details various ways to debug LLDB itself and other LLDB tools. If
+you want to know how to use LLDB in general, please refer to
+:doc:`/use/tutorial`.
+
+As LLDB is generally split into 2 tools, ``lldb`` and ``lldb-server``
+(``debugserver`` on Mac OS), the techniques shown here will not always apply to
+both. With some knowledge of them all, you can mix and match as needed.
+
+In this document we refer to the initial ``lldb`` as the debugger and the
+program being debugged as the debugee.
```

Comment:
The rest of the docs often refer to the program being debugged as the _inferior_. I personally don't mind _debugee_ but inferior would definitely be more consistent/familiar.

Reply:
How about `and the program being debugged as the inferior (sometimes referred to as the debugee).`?

Reply:
Replaced "debugee" with "inferior", since it's only me that seems to use that.

(I always thought "inferior" sounded like a bit severe)